openlr_dereferencer/maps/abstract.py

```diff
@@ -33,10 +33,14 @@ def frc(self) -> FRC:
     def fow(self) -> FOW:
         "Returns the form of way of this line"
 
+    @property
     @abstractmethod
-    def coordinates(self) -> Iterable[Coordinates]:
-        """Returns the shape of the line.
-        Yields GeoCoordinate values."""
+    def shape(self) -> LineString:
```

Comment:
Could we add also an analogous property to `Node` to return a shapely `Point`?  In this case `shape` might not be the best name since a point is shapeless.  I would use `geometry` in both cases for `Line` and `Node` for these properties.

Reply:
Of course, did it now. I also added a tiny 'thing with geometry' base class.